.github/workflows/rust.yml

```diff
@@ -33,7 +33,7 @@ jobs:
 
   # build the library, a compilation step used by multiple steps below
   linux-build-lib:
-    name: AMD64 Debian 10 Rust ${{ matrix.rust }} build libraries
```

Comment:
@kszucs has raised in the past the he would prefer to have the same names as other implementations.

pinging him here.

Reply:
Thanks for the hint @jorgecarleitao  -- will wait for @kszucs  to weigh in

Reply:
I prefer this convention to remain consistent with the rest of the workflows (C++, Python, Ruby, etc.). 
I'm fine with using a different naming convention too if you think that helps distinguishing the builds.

Reply:
I think it will help. Let's try this way and see how we like it. Thanks @kszucs 👍